File: Examen2Trimestre/Python/Rectangulo/TestRectangulo.py
'''
Clase Test Rectángulo que usaremos para probar el funcionamiento de la clase Rectangulo

@author: Cristina Prieto Jalao

'''
from Rectangulo import Rectangulo;

if __name__=="__main__": 
    ancho=int(input('Inserte el ancho del rectángulo: '))
    alto=int(input('Inserte el alto del rectángulo: '))
        
    rectangulo=Rectangulo(ancho, alto)
    print(rectangulo)

File: Examen2Trimestre/Java/Rectangulo/TestRectangulo.java
/**
 * Clase Test Rectángulo que usaremos para probar el funcionamiento de la clase Rectangulo
 * 
 * @author Cristina Prieto Jalao
 */

package Rectangulo;
import java.util.Scanner;

public class TestRectangulo {
  public static void main (String [] args) {
    Scanner s=new Scanner (System.in);
    System.out.print("Inserte el ancho del rectángulo: ");
    int ancho=s.nextInt();
    System.out.print("Inserte el alto del rectángulo: ");
    int alto=s.nextInt();
    
    Rectangulo rectangulo= new Rectangulo(ancho, alto);
    System.out.println(rectangulo);
  }

}
